DelayedClipboard/DelayedClipboardRenderingExplainer.md

```diff
@@ -150,11 +154,72 @@ const clipboardItemInput = new ClipboardItem({['image/png']: blobInput1, ['text/
 navigator.clipboard.write([clipboardItemInput]);
 ```
 
+## What happens to the clipboard data on Tab/Browser Close?
+
+1. Add a `beforeunload` event listener: A web author could choose to `preventDefault` `beforeunload` event and notify the user that there is some data pending to be written into the clipboard. In this event handler, web authors could use a new clipboard API (like `navigator.clipboard.undelayFormats([""]` or something similar) to write the data for the formats that were delay rendered.
+
+e.g.
+```js
+function generateExpensiveCustomFormatBlob() {
+// Produce the expensive data here...
+....
+return new Blob....
+}
+const clipboard_item = new ClipboardItem({
+                'text/html': Promise.resolve(generateExpensiveCustomFormatBlob)
+                });
+navigator.clipboard.write([clipboard_item]);
+
+const beforeUnloadListener = (event) => {
+      event.preventDefault();
+       navigator.clipboard.undelayFormats(['text/html']`);
+      return (event.returnValue = "");
+ };
+```
+
+**Problems with this approach**
+Chrome doesn't allow to customize the message that is shown to the user, so this could be confusing to the user if the default dialog shows up due to formats being delay rendered. Moreover, it's also detrimental to the performance of the site if there is a beforeunload event listener attached.
```

Comment:
```suggestion
Chromium doesn't allow the message that is shown to the user to be customized, so this could be confusing to the user if the default dialog shows up due to formats being delay rendered. Moreover, it's also detrimental to the performance of the site if there is a beforeunload event listener attached.
```

Reply:
Done.